### PR TITLE
chore(js): converts javascript into an AMD module

### PR DESCRIPTION
**Pull request by juho-jaakkola**

### CODE REVIEW

File: views/default/admin/users/upload.php [hunk 1]
```diff
@@ -13,8 +13,7 @@
  * @link http://arckinteractive.com/
  */
 elgg_load_css('upload_users.css');
-elgg_load_js('jquery.form');
```
Comment (juho-jaakkola) [inline views/default/admin/users/upload.php]: I didn't find any other references to jquery.form. Perhaps it has been left here accidentally?

**Comment by hypeJunction** [comment]
Could you also update the manifest with minimum elgg version.

File: views/default/admin/users/upload.php (hunk 1 above)
Comment (hypeJunction) [inline views/default/admin/users/upload.php]: Seems like it.

**Comment by juho-jaakkola** [comment]
Sure

**Pull request merged**